Selenium.Axe/Selenium.Axe/HtmlReport.cs

```diff
@@ -74,16 +74,10 @@ public static void CreateAxeHtmlReport(this IWebDriver webDriver, string destina
         public static void CreateAxeHtmlReport(this IWebDriver webDriver, IWebElement context, string destination, ReportTypes requestedResults = ReportTypes.All)
         {
             var axeBuilder = new AxeBuilder(webDriver);
-            context.CreateAxeHtmlReportFile(axeBuilder.Analyze(context), destination, requestedResults);
+            context.CreateAxeHtmlReport(axeBuilder.Analyze(context), destination, requestedResults);
         }
         
-        
-        public static void CreateAxeHtmlReport(this IWebDriver webdriver, AxeResult results, string destination, ReportTypes requestedResults = ReportTypes.All)
-        {
-            webdriver.CreateAxeHtmlReportFile(results, destination, requestedResults);
-        }
-        
-        private static void CreateAxeHtmlReportFile(this ISearchContext context, AxeResult results, string destination, ReportTypes requestedResults)
+        public static void CreateAxeHtmlReport(this ISearchContext context, AxeResult results, string destination, ReportTypes requestedResults = ReportTypes.All)
```

Comment:
Even this version (where it matches v2.0.1's but with an extra default parameter) [actually represents a binary-breaking change](https://stackoverflow.com/a/23517490) (that whole SO thread is really good, highly recommended reading). The main situation where this matters is if anyone ever has a diamond dependency on the library that gets resolved with a `bindingRedirect`; ideally, you want most libraries to avoid binary-breaking changes in minor/patch versions because users will often expect to be able to use a `bindingRedirect` for consolidating to one copy of a shared dependency. 

Besides this change, I'd recommend keeping all of the old APIs *exactly* as-is and using separate overload with a non-defaulted parameter for the new `ReportTypes` support.

(there do exist scenarios where adding a new overload is also a breaking change, the SO post mentions a few; but in my experience, they're much less common than the diamond dependency + `bindingRedirect` issue)

Reply:
I don't like it, but you being right makes it very hard argue against.

Reply:
Yeah, it definitely looks less pleasant. It's a pretty surprising gotcha with default parameters